### ISSUE TITLE
7.3.7 + opcache segfaults

### ISSUE THREAD

**Issue by remicollet** [remicollet/remirepo#122]
Reported by some users (mail + IRC), encountered with SurgarCRM or GLPI

Could be same than https://bugs.php.net/bug.php?id=78230

Possible workaround for now

* downgrade to 7.3.6
* disable opcache, which have perf impact: `opcache.enable=0`
* disable optimizer, which have much smaller perf impact: `opcache.optimization_level=0`

**Comment by remicollet** [comment]
Also disabling pass 8 SCCP (constant propagation) can be enough
 `opcache.optimization_level=0x7FFFBF7F`

**Comment by remicollet** [comment]
Bug is now fixed upstream
New builds will come very shortly

**Comment by remicollet** [comment]
7.3.7-1 removed from the repository

**Comment by remicollet** [comment]
New build (**7.3.7-3**) done and uploaded.

**Comment by tomsommer** [comment]
Rollback is much appreciated. 

I would propose not pushing releases to repo before it reaches the frontpage if php.net. Rather a day later than earlier. 

I know what just happened is a rare case, but still it caused unneeded disturbance. 

Thanks :)

**Comment by remicollet** [comment]
> I would propose not pushing releases to repo before it reaches the frontpage of php.net. 

Without this push, the problem / regression will stay unfixed, and "official" version will have been released with the bug.

Thanks to "some" reports (mail + irc + forum) we was able to act quickly before official release.

I would rather propose than more people, especially developers or sysadmin with a test env, run the RC version, so such regression will be discovered during RC phase, not after GA.
